### ISSUE TITLE
Scoped packages don't transpile since v6.1.0

### ISSUE THREAD

**Issue by lukeundtrug** [martpie/next-transpile-modules#160]
- [x] I HAVE READ [THE FAQ](https://github.com/martpie/next-transpile-modules#faq) AND MY PROBLEM WAS NOT DESCRIBED THERE
- [x] I WILL GIVE 10$ TO CHARITY IF MY SOLUTION WAS ACTUALLY IN THE README/FAQ

**Are you trying to transpile a local package or an npm package?**
A private npm package 

**Describe the bug**
Our internal dependency doesn't transpile anymore since v6.1.0. No error message is shown. We use Windows machines at our company.

**To Reproduce**
Only reproducable on Windows machines by trying to transpile any scoped module using `/` as a path separator.

**Expected behavior**
Everything should transpile fine, as it did in v6.0.0.

**Setup**

- Next.js version: 10.0.5
- `next-transpile-modules` version: 6.1.0
- Node.js version: 14.15.1
- `npm`/`yarn` version: 6.14.8
- Operating System: Windows 10
- Webpack 4 or 5: 4

```
// next.config.js
const withImages = require('next-images');
const withTM = require('next-transpile-modules')(['@website/shared-resources', '@platform/elements']);

module.exports = withImages(withTM());
```

**Additional context**
I already prepared a PR to try and fix the problem. The fix itself is very trivial. I wrapped the `modulePath` [here](https://github.com/martpie/next-transpile-modules/blob/master/src/next-transpile-modules.js#L127) with `path.normalize` which fixed it for us locally.

I also wrote some unit tests but had to move the matching function out of `withTmInitializer` to be able to unit test it. Would be happy to get feedback on my PR :)

Link to the PR:
[https://github.com/martpie/next-transpile-modules/pull/159](https://github.com/martpie/next-transpile-modules/pull/159)

**Comment by lukeundtrug** [comment]
Fixed in v6.2.0. Thanks @martpie :)

**Comment by martpie** [comment]
no u 😇
